src/ray/gcs/gcs_server/gcs_server_main.cc

```diff
@@ -33,6 +33,7 @@ DEFINE_string(config_list, "", "The config list of raylet.");
 DEFINE_string(redis_password, "", "The password of redis.");
 DEFINE_bool(retry_redis, false, "Whether we retry to connect to the redis.");
 DEFINE_string(node_ip_address, "", "The ip address of the node.");
+DEFINE_string(cluster_id, "", "cluster_id: The unique ID of this cluster.");
```

Comment:
I think it would be nice to mention the current aliasing from session name to cluster id everywhere.

Reply:
Actually I have the following suggestion:
- How about we actually call this `session_name` everywhere in the code.
- Then at the last minute, we record `{"ClusterId": session_name}`.

Then this confusion is really isolated to just the point where it matters--- the metrics recording, and we don't introduce any new terms throughout the codebase.

Alternatively, could we rename cluster_id to session_name?

Reply:
I thought clusterID is a user-facing concept, and setting session_name == clusterID is an implementation detail. If we want to set this at the last minute (when we record "ClustserId"), I feel like it is better just to call it session_name (maybe users already know session_name == cluster ID because it is in the log file). That said, I will go with your second suggestion.